src/pages/signup.tsx

```diff
@@ -16,8 +16,7 @@ const BlobCircle = () => {
 
 type TitledInputProps = {
   title: string;
-  className?: string;
-} & InputHTMLAttributes<HTMLInputElement>;
+} & ComponentProps<typeof Input>;
```

Comment:
Just a q- what made you decide to use ComponentProps instead of the InputHTMLAttributes? Seems like props contains things like refs so maybe we should think about switching all typings to prop-based types?

Reply:
I requested this change in an earlier review. `TitledInput` is an extension of OUR `Input` component, so it should have the same props as our `Input` component.